common/utils/hostname.js

```diff
@@ -40,7 +39,7 @@ function getLocalMachineFullHostname() {
     else
     {
         // On Linux
-        var hostnameCmd = shell.exec('hostname -f');
+	var hostnameCmd = os.hostname(); 
         if (hostnameCmd.code !== 0)
```

Comment:
I see... if you need the FQDN, then I recommend you use:
https://www.npmjs.com/package/node-fqdn